fixbackend/dispatcher/dispatcher_service.py

```diff
@@ -96,18 +120,68 @@ async def process_collect_done_message(self, message: Json, context: MessageCont
             case _:
                 log.info(f"Collect messages: will ignore messages of kine {context.kind}")
 
+    def _collect_progress_hash_key(self, workspace_id: WorkspaceId) -> str:
+        return f"dispatching:collect_jobs_in_progress:{workspace_id}"
```

Comment:
This will bloat the keyspace. Can't we use `dispatching:collect_jobs_in_progress` and use the tenant_it as property in this hash?